### ISSUE TITLE
Why is there no prompt for me to create an account when I install? This will definitely cause me to fail to log in when I open the sentry login page

### ISSUE THREAD

**Issue by leoGaoyaming** [getsentry/self-hosted#2284]
### Problem Statement

Why is there no prompt for me to create an account when I install? This will definitely cause me to fail to log in when I open the sentry login page

**Comment by thingnoy** [comment]
Create user
    $ cd self-hosted-23.7.0
    $ alias sentry="docker-compose run --rm web"
    $ sentry createuser

**Comment by volkansenturk2012** [comment]
When run the docker-compose run --rm web
I show this error 
ERROR: Invalid interpolation format for "platform" option in service "x-sentry-defaults": "${DOCKER_PLATFORM:-}": 
OS: Debian 11
Docker version 24.0.4, build 3713ee1
docker-compose version 1.25.0

**Comment by hubertdeng123** [comment]
Is this a clean install? If the `SKIP_USER_CREATION` is not set an account should be created on install. Otherwise, like the comment mentioned above you can run `docker compose run --rm web createuser`.

**Comment by amato-gianluca** [comment]
I also have the same problem. A clean install (with no command line options) does not prompt nor create any user. Afterwards, I can create a user with <code>docker-compose run --rm web createuser</code>. This is not a big problem in itself, but for the fact that contradicts the documentation.

**Comment by hubertdeng123** [comment]
@amato-gianluca What version are you on? This is potentially a regression since it should prompt user creation upon a clean install

**Comment by ingria** [comment]
I've just installed Sentry and I've also not been prompted for a new account. No idea how to log in.

**Comment by amato-gianluca** [comment]
> @amato-gianluca What version are you on? This is potentially a regression since it should prompt user creation upon a clean install

Version 23.7.1

I tried with several host distributions (Fedora CoreOS 38, CentOS 9 Stream, Ubuntu 22.04) with the same result (at least, with respect to this issue).

**Comment by hubertdeng123** [comment]
I'll attempt this on a fresh install and get back to you all.

**Comment by AlexSSP** [comment]
I have the same problem. And I noticed an error during the installation process and when trying to create a user manually
```
sentry createuser
/usr/local/lib/python3.8/site-packages/memcache.py:1303: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if key is '':
/usr/local/lib/python3.8/site-packages/memcache.py:1304: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if key_extra_len is 0:
```
But on the second manual attempt, the user is created.

**Comment by hubertdeng123** [comment]
Looks like I'm able to reproduce this too. This is definitely a regression.

**Comment by hubertdeng123** [comment]
> /usr/local/lib/python3.8/site-packages/memcache.py:1303: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if key is '':
/usr/local/lib/python3.8/site-packages/memcache.py:1304: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if key_extra_len is 0:
  
  
  You can ignore this, just a warning

**Comment by hubertdeng123** [comment]
Took a look at this again today, this doesn't seem like a trivial fix unfortunately 😕

**Comment by hubertdeng123** [comment]
I think this may have a strong correlation with changes in this file:
https://github.com/getsentry/sentry/blob/1ea6a2ee8702e75b00e08371ccf63daa9e613915/src/sentry/receivers/users.py

**Comment by hubertdeng123** [comment]
Fix for this will be in 23.8.0
